monkestation/code/modules/mob/living/carbon/human/species_type/teratoma.dm

```diff
@@ -0,0 +1,105 @@
+/datum/species/teratoma
```

Comment:
any reason to not make this be a subtype of monkey?

Reply:
because they have several major differences with monkeys, and it'd be annoying working around them

notably: `on_species_gain` mutation stuff and `check_roundstart_eligible`

plus there's likely plenty of things that check `ismonkey(x)` that shouldn't apply to teratomas